packages/docs/verite/patterns/travel-rule.md

```diff
@@ -2,53 +2,104 @@
+
+## The approach
+
+Our design goal in our research and design has been to focus on the privacy problems of discoverability and trust establishment between VASPs and FIs (and perhaps even self-custodied wallets!), imagining an open platform that does not compromise the anonymity guarantees of today's blockchains and marketplaces.
+
+The Verite research group has sketched out a "Verite-maximalist," privacy-preserving architecture for solving these problems by:
+1.) using a decentralized bulletin board discovery mechanism for wallet custodians (or, theoretically, a sufficiently sophisticated VC-enabled self-custody wallet!) to discover one another, then
+2.) querying public, verifiable information about custodians and/or exchanging private verifiable credentials about self-custody counterparties (if accepted by the former) to mutually authenticate a secure channel, and only then
+3.) bootstrapping a mutually-secure connection verifiable credentials exchange about the controllers of both wallets.
+
+Note: The "travel rule" demo in the Verite codebase on github demonstrates one implementation of 1, which we call the "mempool" implementation of a "bulletin board" messaging-based discovery architecture; 2 is still in research phase; and 3 can be achieved today by protocols like TRUST or TRISA, albeit not yet in verifiable credential form.  
+
+### Discovery and Credentials Exchange Sequence
+
+Note, the following diagram reflects the prototype on github, which predates much of the exploratory/theoretical design work of the FATF working group within Verite's open-source development initiative.  The text below presents an expanded, more detailed view; for an understanding of the prototype, refer to the diagram wherever the two differ.
 
 <img src="/img/docs/travel_rule.png" alt="Travel Rule" width="800"/>
 
 ### Part 1: Discovery and Proving Control of Blockchain Addresses
 
+Crucial to the design of our system is some kind of bulletin board system which can be conceived of as a miniature "mempool", i.e. a smart contract that hosts ephemeral messages posted publically for the express purpose of being responded to privately by self-authenticating parties.
+
+Before an intended/proposed transaction is broadcast, the sending and receiving FIs use this chain-agnostic "bulletin board" to publish and observe messages establishing and requesting proof of address control. **These messages contain no customer data and no specific transaction data other than the *recipient* address and sending *custodian's* identity.** 
```

Comment:
I think we can put the hash of an address on the bulletin board for more privacy, instead of just the recipient address

Reply:
good point-- i'll change it for now and let's discuss at next meeting.